### DISCUSSION TITLE
Gyroscope on Technic Hub

### DISCUSSION THREAD

**Discussion by ketchuphed** [pybricks/support#422] [Q&A] [answered]
From looking at the UI in the Control+ app while using the 42011 Liebherr Excavator set controller, the app shows the angle of the machine, the set only has the Technic Hub and the motors, no external gyro. Is this gyro accessible through code somewhere? I don't see reference to it in the docs. https://docs.pybricks.com/en/stable/hubs/technichub.html

**Comment by laurensvalk** [comment, marked as answer]
The gyro and accelerometer are still in beta, but they do work to an extent already.

You are viewing the `stable` version of the docs. You can already see some the new features for those sensors on the `latest` version. You can select it in the bottom left:

![image](https://user-images.githubusercontent.com/12326241/128564903-8f96fb43-4899-4680-bc70-3f1ec0741c88.png)

Then the Technic Hub pages has documentation and working examples for various accelerometer and gyro features.

To try them, make sure to install the firmware and run code via https://beta.pybricks.com/, which gets you the latest.
